src/metabase/api/search.clj

```diff
@@ -336,7 +336,7 @@
 (mu/defn query-model-set :- [:set SearchableModel]
   "Queries all models with respect to query for one result to see if we get a result or not"
   [search-ctx :- SearchContext]
-  (let [model-queries (for [model (:models search-ctx)]
+  (let [model-queries (for [model (search.filter/search-context->applicable-models search.config/all-models search-ctx)]
```

Comment:
By making `models` an argument to `search.filter/search-context->applicable-models` you've now got two "places" models can go. It's a little unnecessary and you have to remember that the models in the search context argument to `search.filter/search-context->applicable-models` is unused.

I think it's tidier to keep `search.filter/search-context->applicable-models` the same and just override `:models` in the search context to be `search.config/all-models` in this specific instance, like so:
```suggestion
  (let [model-queries (for [model (search.filter/search-context->applicable-models (assoc search-ctx :models search.config/all-models)]
```

Reply:
omm, silly me doing that in the first place lol. changed here [77e9f48](https://github.com/metabase/metabase/pull/33038/commits/77e9f482aa3020d4c25ba9df21f50642fd9716d0)